index.js

```diff
@@ -14,6 +14,11 @@ var MIN_PRESERVED_DEPTH = 8
 
 var REPLACEMENT_NODE = '...'
 
+function isError (o) {
+  return o instanceof Error ||
+    /^\[object (Error|(Dom)?Exception)\]$/.test(Object.prototype.toString.call(o))
```

Comment:
This is similiar to the logic in the `iserror` module but fewer bytes. I didn't want to introduce a dependency to this module as it's currently dependency-free.